packages/base/src/UI5Element.js

```diff
@@ -467,6 +485,80 @@ class UI5Element extends HTMLElement {
 		}
 	}
 
+	/**
+	 * @protected
+	 * Creates static area as first child of the body element or returns the existing one.
+	 */
+	_getStaticArea() {
+		const STATIC_AREA_CLASS = "ui5-wc-static-area";
+		let staticArea = document.querySelector(`.${STATIC_AREA_CLASS}`);
+
+		if (staticArea) {
+			return staticArea;
+		}
+
+		// Create static area if it is not present
+		const bodyElement = document.body;
+		staticArea = document.createElement("div");
+		staticArea.classList.add("ui5-wc-static-area");
+		this.staticArea = bodyElement.insertBefore(staticArea, bodyElement.firstChild);
+
+		return this.staticArea;
+	}
+
+	/**
+	 * @public
+	 * Returns reference to the DOM element where the current fragment is added.
+	 */
+	getStaticAreaItemDomRef() {
+		return document.querySelector(`.static-area-item-${this.currentStaticAreaPosition}`).shadowRoot;
```

Comment:
Again, a good candidate for the separate static area class. Ideally UI5Element would have absolutely no knowledge of the StaticArea internals